### PR TITLE
checkbox: allow keyboard toggling

### PR DESCRIPTION
**Pull request by fiws**
pretty simple implementation of keyboard toggling

### CODE REVIEW

File: src/components/mdCheckbox/mdCheckbox.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@ $checkbox-touch-size: 48px;
 
     &:focus {
       outline: none;
+      box-shadow: 0 0 0 2px rgba(#000, .14);
```
Comment (Morgul) [inline src/components/mdCheckbox/mdCheckbox.scss]: Why is this included in this change? It doesn't seem related to keyboard toggling...

Comment (fiws) [inline src/components/mdCheckbox/mdCheckbox.scss]: otherwise there will be no visual indication that the checkbox is focused. keyboard toggling is kinda hard without that ;)

Comment (Morgul) [inline src/components/mdCheckbox/mdCheckbox.scss]: Ah! Ok, that makes sense. :)

Comment (marcosmoura) [inline src/components/mdCheckbox/mdCheckbox.scss]: I think that instead of box-shadow we should highlight the md-ripple element inside. Do you think that you can change that?

Comment (fiws) [inline src/components/mdCheckbox/mdCheckbox.scss]: so, i previously did that (at least made it look like that). 
Thing is: toggling the checkbox will not remove the focus even when clicking, so the active ripple will look rather strange.
